safekeeper/src/timelines_global_map.rs

```diff
@@ -163,14 +165,13 @@ impl GlobalTimelines {
                     {
                         let ttid = TenantTimelineId::new(tenant_id, timeline_id);
                         match Timeline::load_timeline(&conf, ttid) {
-                            Ok(timeline) => {
-                                let tli = Arc::new(timeline);
+                            Ok(tli) => {
                                 let mut shared_state = tli.write_shared_state().await;
                                 TIMELINES_STATE
                                     .lock()
                                     .unwrap()
                                     .timelines
-                                    .insert(ttid, tli.clone());
+                                    .insert(ttid, GlobalMapTimeline::Timeline(tli.clone()));
                                 tli.bootstrap(
```

Comment:
Shoudln't we bootstrap before adding the timeline to the global map or are `GlobalTimelines::get()` callers responsible for checking the status of the timeline? Doesn't make much difference if we don't wait on bootstrap to finish though.

Reply:
Yes, the only thing it does is spawning manager task, so order doesn't matter.